ngraph/test/backend/deformable_convolution.in.cpp

```diff
@@ -0,0 +1,2024 @@
+
+    const Shape inputs_shape{1, 4, 3, 3};
+    const std::vector<float> inputs{1.0f, 2.0f, 3.0f,
+                                    4.0f, 5.0f, 6.0f,
+                                    7.0f, 8.0f, 9.0f,
+                                    10.0f, 11.0f, 12.0f,
+                                    13.0f, 14.0f, 15.0f,
+                                    16.0f, 17.0f, 18.0f,
+                                    19.0f, 20.0f, 21.0f,
+                                    22.0f, 23.0f, 24.0f,
+                                    25.0f, 26.0f, 27.0f,
+                                    28.0f, 29.0f, 30.0f,
+                                    31.0f, 32.0f, 33.0f,
+                                    34.0f, 35.0f, 36.0f};
+
+    const Shape filter_shape{2, 2, 2, 2};
+    const std::vector<float> filter{1.0f, 2.0f,
+                                    3.0f, 4.0f,
+                                    5.0f, 6.0f,
+                                    7.0f, 8.0f,
+                                    -1.0f, -2.0f,
+                                    -3.0f, -4.0f,
+                                    -5.0f, -6.0f,
+                                    -7.0f, -8.0f,
+                                    };
+
+    const Shape offsets_shape{1, 16, 2, 2};
+     const std::vector<float> offsets{// defgroup 1
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     1.0f, 1.0f, 1.0f, 1.0f,
+                                     // defgroup 2
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                     0.0f, 0.0f, 0.0f, 0.0f,
+                                    };
+
+    const Shape outputs_shape{1, 2, 2, 2};
+    const std::vector<float> outputs{500.0f, 234.0f,
+                                     219.0f, 99.0f,
+                                     -1004.0f, -1040.0f,
+                                     -1112.0f, -1148.0f};
+
+    DeformableConvolutionTest(inputs, inputs_shape, offsets, offsets_shape, filter,
+                              filter_shape, outputs, outputs_shape, strides, padding,
+                              dilations, group, deformable_group);
+}
+
+// TODO: deformable convolution atrributes (real offsets)
```

Comment:
TODO: add more test of "deformable convolution atrributes (real offsets)" type

Reply:
Done in [here](https://github.com/openvinotoolkit/openvino/pull/4312/commits/c57481a30be7b9691d32fd94b9a4f9801d93059b).